### DISCUSSION TITLE
Update nested element in a loop ?

### DISCUSSION THREAD

**Discussion by t-prod** [livewire/livewire#4834] [1. Help] [answered]
Hi,

I have a loop like this :

```
@foreach ($lines as $id => $line)
<input wire:model.lazy="lines.{{ $id }}.quantity" class="form-control" type="text">
@endforeach
```

I would like to update this field by using updatedMyFieldName in Livewire how I can do this foreach of nested input ?

Any ideas ?

**Comment by joshhanley** [comment, marked as answer]
@t-prod you can access the key in an updated hook to determine which row has been updated.

Like

```php
public function updatedLines($value, $key)
{
    // If you inspect `$key` here you should see something like `0.quantity` or `1.quantity`, etc. depending on what row has been updated.
}
```

Hope this helps!

**Comment by t-prod** [comment]
@joshhanley thanks I thought I could access directly but by the main name lines it's possible thanks for your answer ;)
